calcom/cal.com
# fix: opt in booking from email

## What does this PR do?

Fixes https://github.com/calcom/cal.com/issues/9207

This is the issue https://github.com/calcom/cal.com/issues/9207#issuecomment-1571975721


## Type of change



- Bug fix (non-breaking change which fixes an issue)

How to test
1) Book a meeting on require confirmation event type
2) Log out of your cal account
3) Now click on confirm button from the email 

you should be directed to page which says
<img width="649" alt="Screenshot 2023-06-07 at 12 55 59 AM" src="https://github.com/calcom/cal.com/assets/53316345/85300da8-5c2e-4735-ab05-86a1a1c33517">
